tests/modules/configs/test_config.py

```diff
@@ -0,0 +1,14 @@
+from src.modules.configs.config import Conf
+
+
+class TestConf:
```

Comment:
Some proposals:
- define some mandatory options and check if they are present in the final_conf
- we should definitely check merging/excluding functionality - if final_conf is according to our expectations
- general approach: try to test each function from source code (as long as it makes sense)
- you can use https://github.com/LegenJCdary/ingeSTation/pull/3/files as a starting point

Reply:
"Good" case example - test should `assert True` for Conf.validate_conf()
```
final = {
	"containers": {
		"sata": {
			"device_path": "/dev",
            "mount_path": "/mnt"
		}
	},
	...
	"exclude": ["ABCD1234", "EFGH5678"],
    "include": []
}
```

Reply:
"Bad" cases examples - test should `assert ValidationError` (or whatever json schema returns when failed):
```
final = {
	"containers": {
		"sata_YYY": {
			"device_path": "",
            "mount_path": ""
		}
	},
	...
	"exclude_XXX": ["ABCD1234", "EFGH5678"],
    "include": []
}
```

```
final = {
	"containers": {
		"sata": {
			"device_path": 369, <- int should not be allowed here (json schema)
            "mount_path": ""
		}
	},
	...
	"exclude": [True, None], <- True and None should not be allowed here
    "include": []
}
```